draft-ietf-jsonpath-base.md

```diff
@@ -697,17 +692,17 @@ in the table below:
 | \\uXXXX            | U+XXXX              | unicode character           |
 {: title="Escape Sequence Replacements" cols="c c"}
 
-The name selector applied to an object
-selects the node of the corresponding member value from it, if and only if that object has a member with that name.
+Applying the `name-selector` to an object node
+selects a member value whose name equals the member name M,
+or selects nothing if there is no such member value.
 Nothing is selected from a value that is not a object.
 
-Note that processing the name selector potentially requires matching strings against
-strings, with those strings coming from the JSONPath and from member
-names and string values in the JSON to which it is being applied.
+Note that processing the name selector requires comparing the member name string M
+with member name strings in the JSON to which the selector is being applied.
 Two strings MUST be considered equal if and only if they are identical
 sequences of Unicode scalar values. In other words, normalization operations
-MUST NOT be applied to either the string from the JSONPath or from the JSON
-prior to comparison.
+MUST NOT be applied to either the member name string M from the JSONPath or to
+the member name strings in the JSON prior to comparison.
```

Comment:
"in the JSON" is (and was) not very preccise.

"in the object node"?

Reply:
I think it's just not member name strings in JSONPath expressions, it's also values in filters, right?

Reply:
Yes, but this probably should be said where those comparisons are described.
(It already gives the algorithm, but doesn't explicitly talk about normalization.